src/Components/DemographicSurvey.js

```diff
@@ -9,9 +11,18 @@ const myCss = {
 
 function DemographicSurvey({ surveyModel }) {
     const [results, setResults] = useState('');
+    const [cacheScreen, setCacheScreen] = useLocalStorage('screenerSurvey', null);
+    const [cacheDem, setCacheDem] = useLocalStorage('demographicSurvey', null);
```

Comment:
Marked the original comment resolved, but wondering if there's a reason we need to cache this? There's no possibility of returning to it if you leave.

Reply:
Yeah, now that I have the cache in here it seems unnecessary and also creates a weird state where you reset the screener survey and have a cached demo survey. So i'm going to remove the cache on the demo survey.